src/main/java/use_cases/add_edit_category_use_case/CategoryID.java

```diff
@@ -0,0 +1,57 @@
+package use_cases.add_edit_category_use_case;
+
+import entities.Category;
+import entities.SessionStorage;
+
+public class CategoryID {
+    private String name;
+    private Object value;
+    private final int monthID;
+
+    private final SessionStorage session;
+    private Category old_category;
```

Comment:
Also to add on to what Kat mentioned, we mentioned in the meeting that this input data class will probably get input from the JComboBox (i.e. drop down of category names), thus you won't get a Category object as input.